test/components/bookmarksToolbarTest.js

```diff
@@ -0,0 +1,61 @@
+/* global describe, it */
+
+const Brave = require('../lib/brave')
+// const { urlInput, saveButton, bookmarksToolbar } = require('../lib/selectors')
+const { bookmarksToolbar } = require('../lib/selectors')
+const settings = require('../../js/constants/settings')
+
+describe('bookmarksToolbar', function () {
+  // function * setup (client) {
+  //   yield client
+  //     .waitUntilWindowLoaded()
+  //     .waitForUrl(Brave.newTabUrl)
+  //     .waitForBrowserWindow()
+  //     .waitForVisible('#window')
+  //     .waitForEnabled(urlInput)
+  // }
+
+  // function * addBookmarkFolder (name) {
+  //   yield this.app.client.moveToObject(bookmarksToolbar)
+  //       .rightClick(bookmarksToolbar)
+  //       .keys('ArrowDown')
+  //       .keys('ArrowDown')
+  //       .keys('ArrowDown')
+  //       .keys('ArrowDown')
+  //       .keys('Enter')
```

Comment:
@darkdh thanks for checking this out- I did end up looking at it more and unfortunately, the menu that pops up is a system popup (not one of our custom rendered context menus, like autofill :disappointed:). I'll remove the commented out tests for now